File: backend/Domain/Domain/Paragraph.cs
using System;

namespace Domain
{
    public class Paragraph
    {
        public Guid Id { get; set; }
        public Document DocumentThatBelongs { get; set; }
        public StyleClass StyleClass { get; set; }
        public Content Content { get; set; }
        public int Position { get; set; }

        public Paragraph()
        {
            DocumentThatBelongs = null;
            StyleClass = null;
            Content = null;
            Position = 0;
        }
    }
}

File: DocManager/src/app/entities/paragraph/paragraph.ts
import { Document } from "../document/document";
import { StyleClass } from "../style-class/style-class";

export class Paragraph {
    
    Id : string;
    Document : Document;
    StyleClass : StyleClass;
    Position : number;

    constructor() {
        this.Id = 'Default';
    }
}